tools/boards.txt.py

```diff
@@ -947,14 +947,14 @@
     ####################### menu.FlashMode
 
     'flashmode_menu': collections.OrderedDict([
+        ( '.menu.FlashMode.dout', 'DOUT' ),
+        ( '.menu.FlashMode.dout.build.flash_mode', 'dout' ),
         ( '.menu.FlashMode.qio', 'QIO' ),
```

Comment:
Would suggest @igrr's ordering here as it is intuitive:  DOUT/DIO/QOUT/QIO.  Most compatible, slowest to least compatible, fastest...

Reply:
Given numbers mentioned somewhere, I agree with that order 😄

Reply:
Order fixed
performance indication in menus
and defines/macro available if needed